### DISCUSSION TITLE
show fetch-only packages on xrepo website

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#2981] [Ideas]
![image](https://user-images.githubusercontent.com/42310255/197995308-a420b321-f6de-40ab-91ea-469dbdedaa14.png)

xrepo上目前搜索不到cuda等fetch-only package，只能搜索到可安装的package。这样很多header-only package的用法没有文档可以说明了。建议加入fetch-only package，根据on_fetch指定其平台，version和install command就不写或者写“install by yourself and xmake will find it from your system”这样的提示

**Comment by waruqi** [comment]
改 web 还是 xrepo search 命令？

**Reply by xq114** [reply]
web，命令现在能搜到

**Reply by waruqi** [reply]
web 暂时不搞，改起来比较折腾，懒得弄。

要改这里，然后又不能影响 ci test，目前是两边复用的。

https://github.com/xmake-io/xmake-repo/blob/31af7eac1f1a86a023f748ffc2d1b605ba0c9194/scripts/packages.lua#L14
